### PR TITLE
feat(docs): use canonical starter pack

### PR DESCRIPTION
**Pull request by syu-w**
Fixes: https://github.com/canonical/craft-store/issues/100
CRAFT-2077

### CODE REVIEW

File: .github/workflows/automatic-doc-checks.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Main Documentation Checks
```
Comment (sergiusens) [inline .github/workflows/automatic-doc-checks.yml:1]: this task we have no where else ....

@lengau ?

Comment (syu-w) [reply .github/workflows/automatic-doc-checks.yml:1]: That one is from the [sphinx-docs-starter-pack](https://github.com/canonical/sphinx-docs-starter-pack/blob/main/.github/workflows/automatic-doc-checks.yml)

Comment (sergiusens) [reply .github/workflows/automatic-doc-checks.yml:1]: this seems like duplication from what we have in tox, so please remove it

Comment (syu-w) [reply .github/workflows/automatic-doc-checks.yml:1]: It seems the commands are different lints. This one check texts / words / links, not the syntax in tox.

Comment (lengau) [reply .github/workflows/automatic-doc-checks.yml:1]: I'm mixed about it. I'd like them to be in tox so, but I'd also like to keep as close to upstream as possible.

Comment (sergiusens) [reply .github/workflows/automatic-doc-checks.yml:1]: ok, let's get this in as is for now, but they names do not follow our convention either.

@lengau can you please open issues to normalize this into tox?

**Review by sergiusens — APPROVED** [review]

File: .github/workflows/automatic-doc-checks.yml (hunk 1 above)
Comment (lengau) [reply .github/workflows/automatic-doc-checks.yml:1]: https://github.com/canonical/craft-store/issues/105

**Pull request merged**